spec/models/ingredient_spec.rb

```diff
@@ -12,4 +21,13 @@
       it { should have_many(:recipes).through(:recipe_ingredients) }
    end
 
+   describe "class methods" do   
+      it "::total_cost" do
+         expect(risotto.ingredients.total_cost).to eq(19)
+      end
+
+      it "::order_alpha" do
+         expect(Ingredient.order_alpha).to eq([ing_1, ing_2, ing_3])
+      end
```

Comment:
To ensure this is actually alphabetizing, I might rearrange your test objects a bit just to verify you aren't getting a false positive since you are already creating these in alphabetical order. They way you test this in the feature test looks great though, so it is still being covered which is key.